tests/explore_test.sh

```diff
@@ -37,6 +38,8 @@ function setUp()
     git add "$file_name" &> /dev/null
     git commit -m "Commit number $commit" &> /dev/null
   done
+  
+  echo "##regex_sample##" > e_grep_test.txt
```

Comment:
To make this test a little bit more robust, I recommend you to add a full text in `tests/samples` and use it for validating your grep function.